python/torch_mlir/dynamo.py

```diff
@@ -67,7 +67,7 @@ def _get_decomposition_table():
         aten.cumsum,
     ]
     # TODO: enable test once 2.1.0 is stable
-    if version.parse(torch.__version__) > version.parse("2.0.1+cpu"):
+    if torch_baseversion() >= (2, 1):
```

Comment:
Doing that, aren't we upgrading from 2.0 to 2.1? (Or was it the goal?)

Reply:
When developing, we are running the nightly torch version (2.1.0+dev...). When testing, ML solutions runs the stable torch version (2.0.1).

Reply:
Before, we would enable the test on 2.0.2, which is likely wrong, as the support only entered 2.1.0.